lib/src/objects/parse_user.dart

```diff
@@ -424,4 +422,11 @@ class ParseUser extends ParseObject implements ParseCloneable {
   }
 
   static ParseUser _getEmptyUser() => ParseUser(null, null, null);
+
+  @override
+  toJson({bool full: false, bool forApiRQ: false}) => <String, String>{
```

Comment:
I'm not sure this will work.

The full boolean is set so that the object can be converted to either a Pointer, or to a JSON string which is stored locally, like the pin function in other Parse SDKs

Reply:
Hello, I did the test and it works on the Pin. 
I did tests using ParseUser and other objects. 

With this change it is possible to save ParseUser as pointer to a class, using 
set <ParseUser>. 

I can send you the test evidence shortly if you want to validate. 

I kept the method signature, but I did not use all the parameters.

Reply:
Sample with code com all possible datatypes:


var parseObject = ParseObject("TestAPI", debug: true);
  parseObject.set<String>("stringName", "Name");
  parseObject.set<double>("doubleNumber", 1.5);
  parseObject.set<int>("intNumber", 0);
  parseObject.set<bool>("boolFound", true);
  parseObject.set<List<String>>("listString", ["a", "b", "c", "d"]);
  parseObject.set<List<int>>("listNumber", [0, 1]);
  parseObject.set<DateTime>("dateTest", DateTime.now());
  parseObject.set<Map<String, dynamic>>(
      "jsonTest", {"field1": "value1", "field2": "value2"});
  parseObject.setIncrement("intNumber2", 2);
  parseObject.setDecrement("intNumber3", 2);
  parseObject.set<ParseGeoPoint>(
      "location", ParseGeoPoint(latitude: -20.2523818, longitude: -40.2665611));
  parseObject.set<ParseUser>("user", user);
  parseObject.set<ParseFile>("fileImage", parseFile);

  apiResponse = await parseObject.save();

  if (apiResponse.success) {